lib/modules/datasource/conan/index.ts

```diff
@@ -143,6 +151,39 @@ export class ConanDatasource extends Datasource {
               }
             }
           }
+
+          if (isArtifactoryServer(rep)) {
+            const conanApiRegexp = /(?<host>.*)\/artifactory\/api\/conan\/(?<repo>[^\/]+)/
+            const groups = url.match(conanApiRegexp)?.groups
+            if (!groups) {
+              return dep
+            }
+            const versioning = 'semver'
```

Comment:
is Conan using strict semver? or can there ba other versioning?

Reply:
Nice question, this is something I don't really like in this code. Conan doesn't strictly use semver, but it is the [default ](https://docs.conan.io/1/creating_packages/define_abi_compatibility.html#versioning-schema). I wouldn't mind changing this, but I couldn't find a way to get the versioning information from the config object to this datasource without breaking the interface, any suggestions?